### ISSUE TITLE
Add alt text for images

### ISSUE THREAD

**Issue by ErinBecker** [carpentries-incubator/rapid-usability-tutorial#7]
It looks like we haven't added alt text for any of the images in this lesson yet. Below is a list of figures, and some suggestions/thoughts on how alt text might be structured for each. The syntax for including alt text for figures, and some guidance on how to write informative alt text, is included in the [figures section](https://carpentries.github.io/sandpaper-docs/episodes.html#figures) of the Workbench documentation.

- [x] [UXR Diagram](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep1_infographic.png)

I don't understand this figure well enough to recommend alternative text. 

- [x] [Avoiding Language screenshot](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep2ex_avoidLang.png)

The to-do and not-to-do examples should be extracted from this image and included in the lesson text, as they are essential for understanding the point being made in the lesson and are not integral to the image. The alt text for this figure can then be something like "Screenshot of Slack messaging interface showing a button labelled 'Reply in thread'."

- [ ] [Multiple means to success](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep2ex_multiMeaning.png)

Expand the text of the example directly preceding this figure to include a description of what is happening in the figure. Something like: 

> success can still be achieved in multiple ways. For example, a user may respond in thread or by starting a new non-threaded message.

The alt text can then be something like "Screenshot of Slack messaging interface showing both threaded and un-threaded responses to a message."

- [x] [Facilitator script](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep2ex_script.png)

The text of the script is not accessible to learners (or instructors) with vision disabilities. This image should be removed and replaced with a text version of the script contents. To make clear that the instructor shouldn't read the script out to the learners, it could be formatted as a callout box. 

- [x] [Anonymous IDs](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep2ex_form.png)

I'm not sure if this image belongs here? It doesn't seem to match the text of the lesson here, and the same image also appears later in the lesson in a place where it seems to fit the text better. 

- [x] [Tracking participation table](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep3_recruit.png)

The content of this table isn't accessible to learners or instructors with vision disabilities. I recommend converting to a markdown table. 

- [x] [Folder structure](https://carpentries-incubator.github.io/rapid-usability-tutorial/fig/ep3_folders.png)

The alt text can be something like "Screenshot of Google Drive folder organisation showing that anonymized data and study findings are stored in shared folders, while raw data is an unshared folder. The participant tracker sheet is unshared and stored in a separate drive from other data."

**Comment by jlcohoon** [comment]
related to #12

**Comment by lucy-dwr** [comment]
I will start on this as part of the STRUDEL dev workshop today! @jlcohoon

**Comment by lucy-dwr** [comment]
Overarching comment: even for people who don’t use assistive technologies, some of the screenshots are blurry, limiting accessibility. Would it be possible to reproduce the screenshots with higher resolution?
